packages/bun-types/redis.d.ts

```
@@ -630,6 +630,115 @@ declare module "bun" {
+     * });
+     *
+     * @note Subscribing disables automatic pipeling, so all commands will be
+     *       received immediately.
+     * @note Subscribing moves the channel to a dedicated subscription state
+     *       which prevents most other commands from being executed until
+     *       unsubscribed. Only @see ping, @see psubscribe, @see punsubscribe,
+     *       @see ssubscribe, @see @subscribe, @see sunsunscribe and @see
+     *       unsubscribe are legal to invoke in a subscribed upon channel.
+     */
+    subscribe(channel: string,
+              listener: (string, string) => void): Promise<void>;
```

Biome (2.1.2)

[error] Duplicate parameter name.

The parameter overrides a preceding parameter by using the same name.

(lint/suspicious/noDuplicateParameters)